src/ItemValue.ts

```diff
@@ -155,7 +155,7 @@ export class ItemValue implements IItemValue {
      * @param value   The desired value to now store.
      */
     public setValue(value: any): void {
-        if (this.transformSet) {
+        if (this.hasOwnProperty("transformSet")) {
```

Comment:
When I did make this change, the places in the file I looked at were checking with `hasOwnProperty` but after just doing another look in the file, `if (this. ...)` is used evenly with `hasOwnProperty`. Should I try to convert to `hasOwnProperty` for checks like these (except for boolean values ofc)  or just revert this change?